app/renderer/polyfills/electron.js

```diff
@@ -3,6 +3,7 @@ import { clipboard, shell, remote, ipcRenderer } from 'electron';
 import log from 'electron-log';
 import settings from 'electron-settings';
 import i18NextBackend from 'i18next-node-fs-backend';
+import fs from 'fs';
```

Comment:
weird does electron make this available to the renderer?

Reply:
@jlipps it does yes! It simplified things quite a bit for me, actually. I'm glad it's there.